mods/cnc/mod.yaml

```diff
@@ -29,13 +18,22 @@ Packages:
 	temperat.mix
 	winter.mix
 	desert.mix
-	snow.mix
 	~movies-gdi.mix
 	~movies-nod.mix
 	~movies.mix
 	~scores.mix
 	~scores2.mix
 	~transit.mix
+	./mods/cnc/bits/snow.mix
```

Comment:
You are right. I even added it myself. https://github.com/OpenRA/OpenRA/commit/a20d1c95c8c6dca8ad431992a5b2b891c97ae4dc